source/javascripts/terminology_tooltip.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+[...document.getElementsByClassName('terminology')].forEach($terminology => {
```

Comment:
Lets not prefix with `$`

Reply:
there is a few other variables that also do this in this file

Reply:
> Lets not prefix with `$`

sure.

just to explain myself: this is a convention kinda taken from jQuery -> vars containing DOM objects are prefixed with `$` -> so it is easy to differentiate them from "normal" objects/var.

like in this case: `$terminology` (the current DOM object) vs `terminology` (the current "term")

Reply:
https://github.com/home-assistant/home-assistant.io/pull/31355/commits/9fc502028c3aac90751ed5a8f5f6042d9c976e5c